extensions/vscode/src/features/nameCasing.ts

```diff
@@ -11,24 +12,25 @@ export async function activate(_context: vscode.ExtensionContext, client: BaseLa
 
 	await client.start();
 
+	const disposes: Disposable[] = [];
```

Comment:
```suggestion
	const disposes: vscode.Disposable[] = [];
```

Reply:
Are there any benefits to using `vscode.Disposable` over directly importing types?

Reply:
I think there are no difference but we are already importing the `vscode` namespace: https://github.com/vuejs/language-tools/pull/3780/files#diff-55aa757c11cb962ca102a6bc49b4de9fddae8a4829f056f30a25246d48834311R1

Reply:
done